.gitignore

```diff
@@ -0,0 +1,5 @@
+venv/
+__pycache__/
```

Comment:
if this was `__pycache__` it should match that directory + all others (e.g. `factory/__pycache__`)